### ISSUE TITLE
Bug 708 again in VS Code Insiders

### ISSUE THREAD

**Issue by byroniac** [microsoft/vscode-js-debug#766]
**Describe the bug**
708 
This is Bug #708 returning, apparently. I had never seen this bug before and it is marked as resolved, but it is occurring in VS Code Insiders. I do not have Node.js installed, and no idea why this is suddenly occurring on my home and work PCs. A possible workaround for me is to go into the @builtin extensions and disable the JavaScript debugger to suppress the error on program launch.

Version: 1.50.0-insider (user setup)
Commit: 403a6895f401c397933ec5c4f7d89432a33509b6
Date: 2020-09-18T05:31:09.978Z
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

[Bug  708](https://github.com/microsoft/vscode-js-debug/issues/708#issue-681246517)

**To Reproduce**
1. Open VS Code Insiders
2. Error pops up immediately, "We did not find "node" on your path, so can not enable auto-attach in your environment" with a link to bug #708 in the Details button.

**Log File** for extension host, as that is the only one I could find with possibly relevant information, sorry:
[2020-09-18 13:54:03.650] [exthost] [info] extension host started
[2020-09-18 13:54:03.660] [exthost] [info] ExtensionService#_doActivateExtension vscode.microsoft-authentication {"startup":false,"extensionId":{"value":"vscode.microsoft-authentication","_lower":"vscode.microsoft-authentication"},"activationEvent":"onAuthenticationRequest:microsoft"}
[2020-09-18 13:54:03.660] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/microsoft-authentication/dist/extension.js
[2020-09-18 13:54:03.667] [exthost] [info] ExtensionService#_doActivateExtension vscode.github-authentication {"startup":false,"extensionId":{"value":"vscode.github-authentication","_lower":"vscode.github-authentication"},"activationEvent":"onAuthenticationRequest:github"}
[2020-09-18 13:54:03.668] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/github-authentication/dist/extension.js
[2020-09-18 13:54:03.673] [exthost] [info] ExtensionService#_doActivateExtension ms-vscode.js-debug {"startup":false,"extensionId":{"value":"ms-vscode.js-debug","_lower":"ms-vscode.js-debug"},"activationEvent":"onDebugDynamicConfigurations"}
[2020-09-18 13:54:03.673] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/ms-vscode.js-debug/src/extension.js
[2020-09-18 13:54:03.748] [exthost] [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch {"startup":true,"extensionId":{"value":"vscode.debug-auto-launch","_lower":"vscode.debug-auto-launch"},"activationEvent":"*"}
[2020-09-18 13:54:03.748] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/debug-auto-launch/dist/extension
[2020-09-18 13:54:03.751] [exthost] [info] ExtensionService#_doActivateExtension vscode.emmet {"startup":true,"extensionId":{"value":"vscode.emmet","_lower":"vscode.emmet"},"activationEvent":"*"}
[2020-09-18 13:54:03.751] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/emmet/dist/node/emmetNodeMain
[2020-09-18 13:54:03.758] [exthost] [info] ExtensionService#_doActivateExtension vscode.git {"startup":true,"extensionId":{"value":"vscode.github","_lower":"vscode.github"},"activationEvent":"*"}
[2020-09-18 13:54:03.758] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/git/dist/main
[2020-09-18 13:54:03.769] [exthost] [info] ExtensionService#_doActivateExtension vscode.merge-conflict {"startup":true,"extensionId":{"value":"vscode.merge-conflict","_lower":"vscode.merge-conflict"},"activationEvent":"*"}
[2020-09-18 13:54:03.769] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/merge-conflict/dist/mergeConflictMain
[2020-09-18 13:54:03.771] [exthost] [info] ExtensionService#_doActivateExtension vscode.search-result {"startup":true,"extensionId":{"value":"vscode.search-result","_lower":"vscode.search-result"},"activationEvent":"*"}
[2020-09-18 13:54:03.771] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/search-result/dist/extension.js
[2020-09-18 13:54:03.817] [exthost] [info] ExtensionService#_doActivateExtension vscode.github {"startup":true,"extensionId":{"value":"vscode.github","_lower":"vscode.github"},"activationEvent":"*"}
[2020-09-18 13:54:03.817] [exthost] [info] ExtensionService#loadCommonJSModule file:///c:/Users/byron/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/extensions/github/dist/extension.js
[2020-09-18 13:54:03.829] [exthost] [info] eager extensions activated

**VS Code Version:** Replace me!

**Additional context**

**Comment by connor4312** [comment]
Thanks for the issue, this duplicates https://github.com/microsoft/vscode/issues/106983 and will be fixed in the next Insiders build.

Until that happens in a few hours, you could ignore this error or manually set `debug.javascript.autoAttachFilter` to `disabled` to avoid this behavior.

**Comment by byroniac** [comment]
> Thanks for the issue, this duplicates [microsoft/vscode#106983](https://github.com/microsoft/vscode/issues/106983) and will be fixed in the next Insiders build.
> 
> Until that happens in a few hours, you could ignore this error or manually set `debug.javascript.autoAttachFilter` to `disabled` to avoid this behavior.

Thanks so much. I did look, but I guess I did not spend enough time looking before I posted.

**Comment by connor4312** [comment]
no worries!

**Comment by byroniac** [comment]
> Thanks for the issue, this duplicates [microsoft/vscode#106983](https://github.com/microsoft/vscode/issues/106983) and will be fixed in the next Insiders build.
> 
> Until that happens in a few hours, you could ignore this error or manually set `debug.javascript.autoAttachFilter` to `disabled` to avoid this behavior.

Just received a new build of Insiders, and this is still not fixed. Will it be fixed soon?

**Comment by connor4312** [comment]
Can you confirm what commit your VS Code is running and whether you have any `debug.javascript.autoAttachFilter` in your settings?

**Comment by byroniac** [comment]
> Can you confirm what commit your VS Code is running and whether you have any `debug.javascript.autoAttachFilter` in your settings?

Never mind, fixed it. On my home PC the installer hung and I had to retry, and then the message went away. On my work PC, the message didn't go away but somehow I had debug.javascript.autoAttachFilter set to onlyWithFlag. But I got some strange error that it could not write to settings. So I nuked my .vscode, .vscode-insiders, and the Roaming app data folders, and started over fresh on everything, and everything is working great now. I didn't have very many settings set so it will be quick to reset them, and I think something weird was (is?) going on with my work machine.
